app/views/api/payments/index.json.jbuilder

```diff
@@ -5,8 +5,8 @@
 json.data @subtransaction_payments, partial: '/api/subtransaction_payments/subtransaction_payment',
```

Comment:
@payments is the name of the class variable being set in the `index` method of `app/controllers/api/payments_controller.rb` that contains all of the `SubtransactionPayment`s to be displayed so that's the one that should be used.

Also, why is there a trailing comma here? That looks worrisome.

Reply:
I fixed the variable name. Sorry for trailing comma, the real problem was an automatic rubocop correction that had added a lot of tabs, so there is a last parameter after the comma trailing that was impossible to see. Please review our contribution again and thank you for your time!!!